src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -468,6 +518,7 @@ module.exports = {
   createSite: trackEvent(createSite, 'createSite'),
   removeSite: trackEvent(removeSite, 'removeSite'),
   createStream: trackEvent(createStream, 'createStream'),
+  editSite: trackEvent(withRunTime(editSite), 'editSite'),
```

Comment:
@c-w we shouldnt have to remove a site to edit a single property. If the createSite fails after the removalal call then the entire pipeline is down.